test/cmdlineTests/yul_verbatim_msize/input.yul

```diff
@@ -0,0 +1,7 @@
+{
+    // The optimizer assumes verbatim could contain msize,
+    // so it cannot optimize the mload away.
+    let x := mload(0x2000)
+    verbatim_0i_0o("")
```

Comment:
Hmm, should we allow empty literals?

Reply:
Why not?

Reply:
What benefit/usecase does the empty literal provide?

Reply:
It can be used as an "optimizer barrier". But regardless of usefulness: I think it is more complicated to disallow it.

Reply:
@chriseth I meant disallowing `verbatim_5i("", a, b, c, d, e)"`

Reply:
I don't think we should add this special case. We cannot check the stack diff in general, why should we check it for the empty literal?

Reply:
Checking the stack diff is not possible because we do not (want) to know what the instructions do. However it is certainly possible to see if a literal is empty or not, given we require it to be a literal.